README.md

```diff
@@ -2,17 +2,21 @@
 Encointer-node branch to build a Testnet with the encointer [community-sidechain](https://github.com/encointer/community-sidechain).
 This node has the Integritee [teerex](https://github.com/integritee-network/pallets/tree/master/teerex) and [sidechain](https://github.com/integritee-network/pallets/tree/master/sidechain) pallets.
 
+## Notes for developers:
+  - The `my-node-runtime` should point to the `encointer-node` [encointer-node](https://github.com/encointer/encointer-node) or you will have a hard time with decoding things.
+  - The type definitions (`ParentchainExtrinsicParams, ParentchainAdditionalParams, ParentchainSignedExtra`) should should be checked that they are compatible with `pallet_asset_tx_payment` (i.e. `ParentchainPlainTip` changed to `ParentchainAssetTip`).
```

Comment:
I do think that these comments are valuable, but I think they would be even more valuable in the worker's fork. Don't you think?

Reply:
You are absolutely right, I will move them, thanks! :100: